database/migrations/2015_10_05_110608_create_messages_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->increments('id');
             $table->text('message');
             $table->boolean('is_seen')->default(0);
+            $table->boolean('is_read')->default(0);
```

Comment:
Please make a new migration file for this changes. otherwise old Talk users can not use this feature.

Reply:
Means this library will now depend on doctrine/dbal and so composer.json must now have this dependency stated, thus old users will need to run composer update. My thoughts.